.github/workflows/global-replicator.yml

```
@@ -95,4 +95,19 @@ jobs:
           committer_username: asyncapi-bot
           committer_email: info@asyncapi.io
           commit_message: "ci: update generic workflows"
-          repos_to_ignore: shape-up-process,glee-hello-world
\ No newline at end of file
+          repos_to_ignore: shape-up-process,glee-hello-world
+
+  replicate_docker_workflows:
+    name: Replicate Docker workflows needed for any project
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Replicating file
+      uses: derberg/copy-files-to-other-repositories@v1.0.0
+      with:
+        github_token: ${{ secrets.GH_TOKEN }}
```

```suggestion
        github_token: ${{ secrets.GH_TOKEN }}
        topics_to_include: docker
```

we want this workflow to be replicated only to repos that have `docker` topic, which is `server-api` repo so far